app/controllers/orga/teams_controller.rb

```diff
@@ -48,7 +47,7 @@ def create
   def update
     @conferences = conference_list
     respond_to do |format|
-      if @team.update_attributes(team_params)
+      if @team.update_attributes(team_params) && terms_accepted?
```

Comment:
You could even go one step further and only validate these field if the user is non-admin 😉 
But that's a bonus and not super necessary for now.